lib/percona_migrator/log_sanitizers/connection_details_sanitizer.rb

```diff
@@ -0,0 +1,21 @@
+module PerconaMigrator
+  module LogSanitizers
+    class ConnectionDetailsSanitizer
```

Comment:
what about renaming it to `PasswordSanitizer`? This makes it clear the class' intent and that to sanitize some other sensible data it's better to create a new one.